cernopendata_client/cli.py

```diff
@@ -55,6 +43,11 @@ def cernopendata_client():
 def get_record(recid, doi, title, output_fields):
     """Get records content by its recid, doi or title filtered."""
     # TODO: Add decorator to require one of (recid, doi or title)
+    if recid is not None and recid <= 0:
+        raise click.BadParameter(
+            'Recid should be a positive integer',
+            param_hint=["--recid"],
+        )
```

Comment:
Note that you have to return and not continue, otherwise some bad URLs are tried:

```console
$ cernopendata-client get-file-locations --recid -1
ERROR: The record id number you supplied is not valid.
404 Client Error: NOT FOUND for url: http://opendata.cern.ch/record/-1
$ cernopendata-client get-file-locations --recid hello
ERROR: The record id number you supplied is not valid.
404 Client Error: NOT FOUND for url: http://opendata.cern.ch/record/hello
```

Reply:
It is returning but I have to add the validation in `get_file_location` method as well.